docs/src/components/productBaseUI/components/BaseButtonDemo.tsx

```diff
@@ -7,7 +7,7 @@ const buttonStyles = `
   font-family: 'IBM Plex Sans', sans-serif;
   font-weight: bold;
   font-size: 0.875rem;
-  background-color: var(--palette-primary);
+  background: var(--muidocs-palette-primary-600);
```

Comment:
I've added some tiny tweaks to the demos, but couldn't solve this one.

Button is using `var(--muidocs-palette-primary)` styles instead of just `var(--palette-primary)`. This is causing a problem with the plain CSS demo, where it pulls the primary blue: 

<img width="612" alt="Screenshot 2024-01-10 at 15 21 04" src="https://github.com/mui/material-ui/assets/37222944/ad5a7930-f033-4527-94b8-ef772077d5e8">


However, when I change the imports to `var(--palette-primary)` it ignores it:

<img width="629" alt="Screenshot 2024-01-10 at 15 26 35" src="https://github.com/mui/material-ui/assets/37222944/cd2bcd43-cb5a-4dbf-a8fc-546a1477482b">